code/controllers/configuration/entries/general.dm

```diff
@@ -62,7 +62,7 @@
 
 /datum/config_entry/flag/log_prayer // log prayers
 
-/datum/config_entry/flag/log_law // log lawchanges
+/datum/config_entry/flag/log_silicon // log silicons
```

Comment:
we have a system for moving configs over to new configs while warning server owners:

see here: for an example: https://github.com/tgstation/tgstation/blob/fa8199a01a12b61606e3613e627327eecb5999ea/code/controllers/configuration/entries/game_options.dm#L224-L243